index.html

```diff
@@ -4987,8 +4977,20 @@ <h2 id="AudioWorkletGlobalScope">
             This special execution context is designed to enable the
             generation, processing, and analysis of audio data directly using
             JavaScript in the audio <a>rendering thread</a>. The user-supplied
-            script code is evaluated in this scope to define an
-            <a>AudioWorkletProcessor</a>.
+            script code is evaluated in this scope to define one or more
+            <a>AudioWorkletProcessor</a> subclasses, which in turn are used to
+            instantiate <a>AudioWorkletProcessor</a>s, in a 1:1 association
+            with <a>AudioWorkletNode</a>s in the main scope.
+          </p>
+          <p>
+            At least one <a>AudioWorkletGlobalScope</a> exists for each
```

Comment:
I don't think it makes any difference how many AWGSs are created, since if there is more than one, the effects of multiple AWGSs must not be observable. We are not mandating any sort of parallel processing here, but also not ruling it out provided that its effects are unobservable (which is important for the future).

However I believe this is in fact normative language because of the requirement of "At least one". What _is_ observable is that at least one AWGS exists, allowing processing to take place at all.